script/training_db_sync.rb

```diff
@@ -106,7 +106,7 @@ def call
     Kernel.puts "Copying pw reset script to #{destination} ==> #{copy_cmd}"
     CmdRunner.run(copy_cmd)
 
-    reset_cmd = %(cf ssh beis-roda-#{destination} -c "bin/rails runner #{pw_reset_script}")
+    reset_cmd = %(cf ssh beis-roda-#{destination} -c 'cd /app; PATH=$PATH:/usr/local/bin bin/rails runner #{pw_reset_script}')
```

Comment:
From the commit message, it sounds like single quotes should make it load the correct path, so what's the (extra) benefit of the explicit path?

Also, are there any docs for the single versus double quote distinction that could be linked to in the commit? The commit is a good explainer of why, but it feels like a pretty unusual distinction so some docs might be nice!

Reply:
Yeah, I (lazily) put both in, but we don't need the single quotes here so have put them back to double quotes - I think the discovery that they use your local PATH was a bit of a red herring that wasn't actually required. I'm still not 100% sure why the PATH isn't loaded properly when running `ssh -c` but will see what I can find here.